ui_components/src/sidebar/Sidebar.js

```diff
@@ -137,10 +141,10 @@ function Sidebar({ links, navigationAPIData, userAPIData }) {
                 regenTime={regenTime}
                 regenOffset={regenOffset}
             />
-            {displaySection(userMenu, "Player Menu", toggleNav)}
-            {displaySection(activityMenu, "Action Menu", toggleNav)}
-            {displaySection(villageMenu, "Village Menu", toggleNav)}
-            {staffMenu.length ? displaySection(staffMenu, "Staff Menu", toggleNav) : null}
+            {displaySection(userMenu, "Player Menu", toggleNav, playerSettings.sidebar_collapse, parentClick)}
```

Comment:
When the function is pulled out to a component, these can be

```js
<DisplaySection
   sectionData={userMenu}
   title="Player Menu"
   toggleNav={toggleNav}
   sidebarCollapsed={playerSettings.sideBar_collapse}
   parentClick={parentClick}
/>
```

Reply:
I agree, need to take a pass at the original React work I did using functions instead of functional components.
Will do this as a follow-up.